modules/nav-menu.js

```diff
@@ -0,0 +1,39 @@
+// Navigation menu
+/* eslint-disable */
```

Comment:
- [ ] Linters help us write code based on pre-defined standards. Except for third-party files (like luxon), avoid disabling linters. Kindly enable linters here and fix the resulting errors.